sample.env

```diff
@@ -7,5 +7,4 @@ CONFIG_BACKEND_URL="https://api.rudderstack.com"
 REDIS_HOST = localhost
 REDIS_PORT = 6379
 REDIS_PASSWORD = 123
-REDIS_USERNAME = abc
```

Comment:
this field is required

Reply:
@shrouti1507 did you delete this?

Reply:
don't think so. Some merge results I guess. Putting it back.